discovery-provider/alembic/env.py

```diff
@@ -71,6 +83,7 @@ def run_migrations_online():
     )
 
     with connectable.connect() as connection:
+        connection.execute(kill_running_queries_sql)
```

Comment:
should wrap in a `sqlalchemy.text` call ([example](https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/alembic/versions/38642fb2948d_add_pg_triggers.py#L30))

otherwise any `%` will cause an obscure sqlalchemy error to be thrown.

Reply:
%% fixed this for me, but i like sqlalchemy.text more